drake/systems/plants/RigidBodyTree.cpp

```diff
@@ -511,7 +513,7 @@ bool RigidBodyTree::collisionDetect(const KinematicsCache<double>& cache,
                                     VectorXd& phi, Matrix3Xd& normal,
                                     Matrix3Xd& xA, Matrix3Xd& xB,
                                     vector<int>& bodyA_idx,
-                                    vector<int>& bodyB_idx, bool use_margins) {
+                                    vector<int>& bodyB_idx, bool use_margins) { //use_margins = true by default in constructor
```

Comment:
This comment has a maintainability problem. It may be true currently, but if it were ever changed in the header (by some unknown future programmer) there would be no way to ensure that the comment was properly updated. It would be better to leave it off and let people look at the declaration if they care about the default. (Easy for those of us with a decent IDE!)